cmd/builder/internal/command.go

```diff
@@ -82,6 +83,7 @@ configuration is provided, ocb will generate a default Collector.
 	// the distribution parameters, which we accept as CLI flags as well
 	cmd.Flags().BoolVar(&cfg.SkipGenerate, skipGenerateFlag, false, "Whether builder should skip generating go code (default false)")
 	cmd.Flags().BoolVar(&cfg.SkipCompilation, skipCompilationFlag, false, "Whether builder should only generate go code with no compile of the collector (default false)")
+	cmd.Flags().BoolVar(&cfg.SkipGenerateMain, skipGenerateMainFlag, false, "Whether builder should skip generating main.go, main_others.go and main_windows.go (default false)")
```

Comment:
Do we need a flag? Why?

Reply:
> We have the need to generate list of components in tooling adjacent to the collector, such as configschema.

I also need this for distros where main.go is managed manually.